### PR TITLE
Translate Karmada API index description

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation
/kind feature

**What this PR does / why we need it**:

Translate Karmada API index description

**Which issue(s) this PR fixes**:
Part of #483 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by XiShanYongYe-Chang** [comment]
preview:

image

**Comment by XiShanYongYe-Chang** [comment]
/cc @RainbowMango @chaosi-zju 
Can you help take a review?

File: i18n/zh/docusaurus-plugin-content-docs/current.json [hunk 1]
```diff
@@ -49,7 +49,10 @@
   },
   "sidebar.docs.category.Karmada API": {
     "message": "Karmada API",
-    "description": "Karmada API 文档"
+    "description": "The label for category Karmada API in sidebar docs"
+  },
+  "sidebar.docs.category.Karmada API.link.generated-index.description": {
+    "message": "Karmada API 是通过 RESTful 接口提供 Karmada 功能服务并负责 Karmada 状态存储的应用程序。\n\nKarmada 资源和\"意向记录\"都是作为 API 对象储存的，并可以通过调用 RESTful 风格的 API 进行修改。 API 允许以声明方式管理配置。 用户可以直接和 Karmada API 交互，也可以通过 kubectl 这样的工具进行交互。 核心的 Karmada API 是很灵活的，可以扩展以支持定制资源。"
```
Comment (chaosi-zju) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:55]: this `\n\n` is useless

Comment (chaosi-zju) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:55]: kubectl -> karmadactl

Comment (chaosi-zju) [inline i18n/zh/docusaurus-plugin-content-docs/current.json:55]: may there doesn't need a empty space behind `。`

I mean 

* `配置。 用户` ->  `配置。用户`
* `交互。 核心` -> `交互。核心`

Comment (XiShanYongYe-Chang) [reply i18n/zh/docusaurus-plugin-content-docs/current.json]: Here we should use kubectl.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**